examples/configures/MTR_ECFP/PDBbind.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Where did you use this?

Reply:
I put these configuration files in so that the benchmarks can be re-run without hyperparam optimization. I can remove if needed.